app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
```

Comment:
Oh, I see.  `belongs_to` is much nicer than manual finding by foo_id!
- src_container_node has other logic to workaround #9600.  But I can move the workaround to where container_node_id is set, and actually I'll split that to its own tiny PR.
- I felt it's important to log failed lookups.  I'll re-check whether it's logged elsewhere and possibly do a separate PR to add logging.
  [Motivation: I expect some of these lookups to fail when the event refers to entity not yet seen in refresh.  Currently such events won't get re-connected even when we later see it — in those cases policies just won't run.  So I wanted this situation to at least be explained in the log.]

Q: I originally picked `target=src_container_...` names in parallel to `src_vm` or something.
But now I realize vms and friends have `src_...` vs `dest_...` which containers don't.
Should I keep to the src_container_... aliases or drop it and just write to `target=container_...` in the automate actions?